scripts/install_atos.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+
+# Check if called without arguments
+if [ $# -eq 0 ]; then
+    echo "Don't call this file directly, use setup_atos.sh instead."
+    echo "Usage: ./install_deps.sh <path to ATOS git repo>"           
+    exit 0
+fi
+
+# Take the first argument as the PATH to the ATOS git repo
+ATOS_REPO_PATH="$1"
+source "${ATOS_REPO_PATH}/scripts/install_functions.sh"
```

Comment:
check_command_failed could be added here to abort the program if an incorrect ATOS_REPO_PATH was input